packit/api.py

```diff
@@ -452,15 +456,28 @@ def create_srpm(
         """
         self.up.run_action(actions=ActionName.post_upstream_clone)
 
-        source_dir = self.up.prepare_upstream_for_srpm_creation(
-            upstream_ref=upstream_ref
-        )
+        try:
+            source_dir = self.up.prepare_upstream_for_srpm_creation(
+                upstream_ref=upstream_ref
+            )
+        except Exception as ex:
+            raise PackitSRPMException(
+                f"Preparing of the upstream to the SRPM build failed: {ex}", ex
+            )
```

Comment:
How is it different from the [raise-from](https://stackoverflow.com/questions/24752395/python-raise-from-usage) ?
```python
raise PackitSRPMException(f"Preparing of the upstream to the SRPM build failed: {ex}") from ex
```

Reply:
Good to know.

Reply:
Changed.